### PR TITLE
Make DROP statements consistent

### PR DESCRIPTION
**Pull request by c-w**
No functional change.

### CODE REVIEW

File: ops/storage-ddls/cassandra-setup.cql [hunk 1]
```diff
@@ -57,8 +57,6 @@ CREATE TABLE trustedsources (
    PRIMARY KEY (pipelinekey, externalsourceid, sourcetype, rank)
 );
 
-DROP TABLE IF EXISTS events;
-DROP TYPE IF EXISTS features;
```
Comment (jcjimenez) [inline ops/storage-ddls/cassandra-setup.cql:61]: These drop statements need to be here. If you put them next to their `CREATE TABLE` counterparts, the `DROP TABLE` of their referencing table(s) will fail. For instance, the `DROP TABLE features` will fail if we don't re-arrange the order in which it's dependency tables are added.

Having said that, I agree that having three seemingly-random DROP TABLE statements together looks weird. Can I suggest we move the `CREATE TABLE` AND `CREATE TYPE` statements instead?

**Review by jcjimenez — COMMENTED** [review]
I propose we move the `CREATE` statements instead of the `DROP` ones, what do you think?

**Review by erikschlegel — APPROVED** [review]
LGTM

File: ops/storage-ddls/cassandra-setup.cql (hunk 1 above)
Comment (c-w) [reply ops/storage-ddls/cassandra-setup.cql:61]: Good catch, moved everything together in f9c7fdc. Ran it against cqlsh and seems to work.

**Pull request merged**